src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -946,6 +946,10 @@ bool Unit::HasBreakableByDamageCrowdControlAura(Unit* excludeCasterChannel) cons
         }
     }
 
+    // check to see if victim is sitting
+    if (victim->GetStandState())
```

Comment:
should this affect already players, in case some script set a stand state to a boss during a fight ?

Reply:
@aurelio4 do you have a reply to my comment above ?

Reply:
Yeah, it's safer to check for player only